Surface some errors in communication

jasonzurita opened the pull request:
Alert error will be shown when reachability is false or if we can't encode the info to send.

Handling the error completion handler when sending a message is something for the future since that is a bit trickier.

jasonzurita committed:
feat: surface some errors in communication

Alert error will be shown when reachability is
false or if we can't encode the info to send.

Handling the error completion handler when
sending a message is something for the future
since that is a bit trickier.

FlippingLeaf commented:
<!--
  0 Errors
  1 Warning: <a href="https://github.com/ja...
  0 Messages
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/jasonzurita/BabyPatterns/blob/206e71984511042b2d620309316832e662fdfa36/Watch Extension/AppState.swift#L36">Watch Extension/AppState.swift#L36</a> - Function body should span 40 lines or less excluding comments and whitespace: currently spans 42 lines<br />
<code>function_body_length</code> <code>AppState.swift:36</code></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

jasonzurita committed:
chore: disable one off linter rule

function body length is okay here
since this is a switch statement with clear
branching logic as a result
